po/pl.po

```diff
@@ -2062,6 +2179,9 @@ msgstr "na licencji GNU GPLv2 (lub nowszej)"
 msgid "menu"
 msgstr "meni"
 
+msgid "Metronome"
+msgstr "Taktomierz"
```

Comment:
```suggestion
msgstr "Metronom"
```

I've never ever heard any musician use the name "taktomierz", even though it apparently is a valid one. Still a valuable thing to learn :D